eth/main.cpp

```diff
@@ -106,74 +108,74 @@ void help()
 	
-		<< "    --public-ip <ip>  Force advertised public ip to given (default: auto)." << endl
+		<< "    --public-ip <ip>  Force advertised public IP to the given IP (default: auto)." << endl
 		<< "    --listen-ip <ip>(:<port>)  Listen on the given IP for incoming connections (default: 0.0.0.0)." << endl
 		<< "    --listen <port>  Listen on the given port for incoming connections (default: 30303)." << endl
-		<< "    -r,--remote <host>(:<port>)  Connect to remote host (default: none)." << endl
-		<< "    --port <port>  Connect to remote port (default: 30303)." << endl
-		<< "    --network-id <n> Only connect to other hosts with this network id." << endl
+		<< "    -r,--remote <host>(:<port>)  Connect to the given remote host (default: none)." << endl
+		<< "    --port <port>  Connect to the given remote port (default: 30303)." << endl
+		<< "    --network-id <n>  Only connect to other hosts with this network id." << endl
 		<< "    --upnp <on/off>  Use UPnP for NAT (default: on)." << endl
 
-		<< "    --peerset <list>  Space delimited list of type type:publickey@ipAddress[:port]" << endl
+		<< "    --peerset <list>  Space delimited list of peers; element format: type:publickey@ipAddress[:port]." << endl
 		<< "        Types:" << endl
-		<< "        default		Attempt connection when no other peers are available and pinning is disable." << endl
+		<< "        default		Attempt connection when no other peers are available and pinning is disabled." << endl
 		<< "        require		Keep connected at all times." << endl
 // TODO:
 //		<< "	--trust-peers <filename>  Space delimited list of publickeys." << endl
 	
-		<< "    --no-discovery  Disable Node discovery, implies --no-bootstrap." << endl
+		<< "    --no-discovery  Disable node discovery, implies --no-bootstrap." << endl
 		<< "    --pin  Only accept or connect to trusted peers." << endl
 		<< "    --hermit  Equivalent to --no-discovery --pin." << endl
-		<< "    --sociable  Forces discovery and no pinning." << endl
+		<< "    --sociable  Force discovery and no pinning." << endl
 		<< endl;
 	MinerCLI::streamHelp(cout);
 	cout
 		<< "Client structured logging:" << endl
-		<< "    --structured-logging  Enable structured logging (default output to stdout)." << endl
+		<< "    --structured-logging  Enable structured logging (default: output to stdout)." << endl
 		<< "    --structured-logging-format <format>  Set the structured logging time format." << endl
 		<< "    --structured-logging-url <URL>  Set the structured logging destination (currently only file:// supported)." << endl
 		<< endl
 		<< "Import/export modes:" << endl
-		<< "    --from <n>  Export only from block n; n may be a decimal, a '0x' prefixed hash, or 'latest'." << endl
-		<< "    --to <n>  Export only to block n (inclusive); n may be a decimal, a '0x' prefixed hash, or 'latest'." << endl
+		<< "    --from <n>  Export only from block n; n may be a decimal, a '0x' pre-fixed hash, or 'latest'." << endl
```

Comment:
"prefixed" (no hyphen) is fine. see [definition of "to prefix" verb](http://www.thefreedictionary.com/prefix).